examples/collector-exporter-node/start.js

```diff
@@ -1,11 +1,13 @@
 'use strict';
 
 const opentelemetry = require('@opentelemetry/api');
+// const { ConsoleLogger,  LogLevel} = require('@opentelemetry/core');
```

Comment:
Can we remove this?

Reply:
I'm keeping them for easier debugging only so that I can see which option has been chosen how the spans looks like etc. and recently have been using this quite heavily. Although not strong against not removing if you think they shouldn't be. But this is just an example mainly for devs so I guess it might be useful to. WDYT ?

Reply:
@mayurkale22 
https://github.com/open-telemetry/opentelemetry-js/blob/905a5d6503a53e39875342f7d7066da6bfd06aa7/examples/collector-exporter-node/start.js#L10
and here
https://github.com/open-telemetry/opentelemetry-js/blob/905a5d6503a53e39875342f7d7066da6bfd06aa7/examples/collector-exporter-node/start.js#L11

Reply:
Think this is fine since it is an example.

Reply:
@mayurkale22 ok to merge?

Reply:
Sure.